### PR TITLE
faq: update TiDB Lightning FAQ

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? 

统一文档部分 FAQ

### Which version does your change affect? 

v2.1/v3.0/dev

### CODE REVIEW

**Comment by lance6716** [comment]
PTAL @lilin90

**Comment by lilin90** [comment]
@yikeke Please help review this PR. Thanks!

File: dev/faq/tidb-lightning.md [hunk 1]
```diff
@@ -77,25 +77,29 @@ sql-mode = ""
...
+- 手动部署：如果 `tidb-lightning` 正在前台运行，可直接按 <kbd>Ctrl</kbd>+<kbd>C</kbd> 退出。否则，可通过 `ps aux | grep tidb-lightning` 获取进程 ID，然后通过 `kill -2 «pid»` 结束进程。
 
-## 进程在服务器上运行，进程莫名其妙地就退出了，是怎么回事呢？
+## `tidb-lightning` 在服务器上运行，进程莫名其妙地退出了，是怎么回事呢？
 
-这种情况可能是启动方式不正确，导致因为收到 SIGHUP 信号而退出，此时 `tidb-lightning.log` 通常有这幺一行日志：
+这种情况可能是启动方式不正确，导致收到 SIGHUP 信号而退出。此时 `tidb-lightning.log` 通常有如下日志：
 
 ```
 2018/08/10 07:29:08.310 main.go:47: [info] Got signal hangup to exit.
```
Comment (kennytm) [inline dev/faq/tidb-lightning.md]: Better change this line to the new log format.

File: v3.0/faq/tidb-lightning.md [hunk 2]
```diff
@@ -78,25 +78,29 @@ sql-mode = ""
...
+- 手动部署：如果 `tidb-lightning` 正在前台运行，可直接按 <kbd>Ctrl</kbd>+<kbd>C</kbd> 退出。否则，可通过 `ps aux | grep tidb-lightning` 获取进程 ID，然后通过 `kill -2 «pid»` 结束进程。
 
-## 进程在服务器上运行，进程莫名其妙地就退出了，是怎么回事呢？
+## `tidb-lightning` 在服务器上运行，进程莫名其妙地退出了，是怎么回事呢？
 
-这种情况可能是启动方式不正确，导致因为收到 SIGHUP 信号而退出，此时 `tidb-lightning.log` 通常有这幺一行日志：
+这种情况可能是启动方式不正确，导致收到 SIGHUP 信号而退出。此时 `tidb-lightning.log` 通常有如下日志：
 
 ```
 2018/08/10 07:29:08.310 main.go:47: [info] Got signal hangup to exit.
```
Comment (kennytm) [inline v3.0/faq/tidb-lightning.md]: ditto

**Comment by lance6716** [comment]
Updated log format on branch later than v3.0 @kennytm

**Pull request merged**